assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/TicketAssignmentsManagerModal.tsx

```diff
@@ -1,32 +1,39 @@
 import React from 'react';
 import { __ } from '@wordpress/i18n';
+import { useDisclosure } from '@chakra-ui/core';
 
+import { Modal } from '@infraUI/layout/modal';
+
+import ErrorModal from './ErrorModal';
 import TicketAssignmentsManager from './TicketAssignmentsManager';
 import useCancelButtonProps from '../buttons/useCancelButtonProps';
 import useSubmitButtonProps from '../buttons/useSubmitButtonProps';
 import { useTAMContext } from '../context';
-import { Modal } from '@infraUI/layout/modal';
 
 import './styles.scss';
 
 const TicketAssignmentsManagerModal: React.FC = () => {
 	const { onCloseModal, title } = useTAMContext();
-	const cancelButtonProps = useCancelButtonProps();
+	const { isOpen, onOpen: onOpenErrorModal, onClose } = useDisclosure();
```

Comment:
initially I was thinking to push this in `useTAMContext`, but later decided that for this simple state management `useDisclosure` is suffice